virt/node_maintenance/virt-preventing-node-reconciliation.adoc

```diff
@@ -0,0 +1,16 @@
+[id="virt-using-skip-node"]
+= Preventing node reconciliation
+include::modules/virt-document-attributes.adoc[]
+include::modules/common-attributes.adoc[]
+:context: virt-using-skip-node
+
+Use `skip-node` annotation to prevent the `node-labeller` from reconciling a node.
+
+toc::[]
+
+include::modules/virt-using-skip-node.adoc[leveloffset=+1]
+
+[id="additional-resources"]
```

Comment:
This needs to be a unique ID, so I've been using `[id="additional-resources_{context}"]`.